src/pages/feature/heehyun/Login/Login.js

```diff
@@ -0,0 +1,130 @@
+import React, { useState } from 'react';
+import './Login.scss';
+import logo1 from './Logo.png';
+import logo2 from './logo_wecode.png';
+import { useNavigate } from 'react-router-dom';
+
+const Login = () => {
+  // 라우터를 통해 네비게이트 함수 활성화
+  const navigate = useNavigate();
+
+  // 이메일, 비밀번호 저장할 State 사용
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+
+  // 이메일, 비밀번호를 객체로 저장할 State 사용
+  // const [user, setUser] = useState([]);
+
+  // 이메일, 비밀번호를 저장할 함수 사용
+  const setChangeEmail = (event) => {
+    setEmail(event.target.value);
+  };
+
+  const setChangePassword = (event) => {
+    setPassword(event.target.value);
+  };
+
+  // 버튼을 활성화시킬 수 있는 active 변수 사용
+  const [active, setActive] = useState(false);
+
+  // 로그인 버튼을 눌렀을때 이메일과 비밀번호를 저장하면서 동시에 메인 페이지 이동
+  const goToMain = () => {
+    fetch('http://10.58.52.224:8000/users/login', {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json;charset=utf-8',
+      },
+      body: JSON.stringify({
+        // id: user.length + 1,
+        email: email,
+        password: password,
+        // nickname: nickname,
+        // mobileNumber: mobileNumber,
+      }),
```

Comment:
객체의 key와, value로 사용할 변수명이 같을 때는 아래처럼 축약 할 수 있습니다.
```suggestion
      body: JSON.stringify({
        email,
        password,
      }),
```
또한 email과 password를 하나의 state로 관리한다면 다음과 같이 축약할 수 있습니다.
```suggestion
const. [state,setState] = useState({
email:"",
password:""
})
.
.
.
  body: JSON.stringify(state),
```